microbit-common/Cargo.toml

```diff
@@ -34,7 +34,8 @@ version = "0.12.1"
 
 [dependencies.nrf52833-hal]
 optional = true
-version = "0.12.1"
+version = "0.13.0"
```

Comment:
I think it would be good to keep the two nrf-hal dependencies at the same version. Would it be possible to bump `nrf51-hal` to `0.13.0` as well?

Reply:
Done!